lir/data/models.py

```diff
@@ -144,6 +157,34 @@ def check_matching_shapes(self) -> Self:
         return self
 
 
+class PairData(FeatureData):
```

Comment:
Misschien is `PairedFeatureData` een betere naam, omdat het nog steeds gaat over de features van de instances zelf, en niet van de paren.